languages/csharp/exercises/concept/object-initializers/ObjectInitialization.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+
+//**** please do not modify the FacialFeatures class ****
+public class FacialFeatures
+{
+    public string EyeColor { get; set; }
+    public decimal PhiltrumWidth { get; set; }
+
+    public FacialFeatures()
```

Comment:
We could consider removing the constructor here, which is how I often see classes declared that one wants to initialize using an object initializer. This comment also applies to the `Identity` class.

Reply:
done